recipes/mini-racer/recipe.yaml

```diff
@@ -0,0 +1,67 @@
+schema_version: 1
+
+context:
+  name: mini-racer
+  version: 0.12.4
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/mini_racer-${{ version }}.tar.gz
+  sha256: 84c67553ce9f3736d4c617d8a3f882949d37a46cfb47fe11dab33dd6704e62a4
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  dynamic_linking:
+    binary_relocation: ${{ false if osx else true }}
+
+
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler("cxx") }}
+    - if: build_platform != target_platform
+      then: cross-python_${{ target_platform }}
+    - if: build_platform != target_platform
+      then: python
+    - if: not win
+      then: pkg-config
+    - six
+    - gsutil
+  host:
+    - python
+    - hatchling
+    - hatch-fancy-pypi-readme
+    - hatch-mkdocs
+    - pip
+    - httplib2 <0.30.0  #Apparent issues with httplib2
+    - packaging
+    - six
+  run:
+    - python
+
+tests:
+  - python:
+      imports:
+        - py_mini_racer
+  - requirements:
```

Comment:
this is made redundant by the above `python` import test, which can include an explicit `pip_check: true`